jax/api.py

```diff
@@ -52,17 +53,11 @@
 from .util import (unzip2, unzip3, curry, partial, safe_map, safe_zip,
                    WrapHashably, Hashable, prod, split_list)
 from .lib import xla_bridge as xb
-from .lib.xla_bridge import (device_count, local_device_count, devices, local_devices,
-                             host_id, host_ids, host_count)
-from .abstract_arrays import ConcreteArray, ShapedArray, raise_to_shaped
-from .interpreters import partial_eval as pe
-from .interpreters import xla
-from .interpreters import pxla
-from .interpreters import ad
-from .interpreters import batching
-from .interpreters import parallel
-from .interpreters import masking
-from .interpreters.masking import shapecheck, ensure_poly
+from .lib.xla_bridge import (device_count, local_device_count, devices,
+                             local_devices, host_id, host_ids, host_count)
+from .abstract_arrays import ConcreteArray, ShapedArray, raise_to_shaped, \
+  eval_polymorphic_shape, Poly, Mon
+from .interpreters import partial_eval as pe, xla, pxla, ad, batching, parallel
```

Comment:
Can you avoid changes to unrelated import statements?

Reply:
Sure. I reverted import changes to a minimum.